maps/templates/orbital/orbital_templates.dm

```diff
@@ -1,11 +1,23 @@
 /datum/map_template/mecha
-	name = "Mecha Drop"
+	name = "Combat Mecha Drop"
 	mappath = 'maps/templates/orbital/combat-mecha.dmm'
 
+/datum/map_template/mecha/heavy
+	name = "Heavy Mecha Drop"
+	mappath = 'maps/templates/orbital/heavy-mecha.dmm'
+
+/datum/map_template/mecha/light
```

Comment:
My preference would be to make the light the default and then go from there with subpaths